### ISSUE TITLE
docker vllm测试，输出token全部是151872

### ISSUE THREAD

**Issue by zhujlfree** [QwenLM/Qwen2.5-Omni#141]
2025.4.2拉取的docke镜像，跑了下面的vllm测试，但是输出的token全是151872？能帮忙看看是为什么吗？谢谢。

import os
os.environ["CUDA_VISIBLE_DEVICES"] = "7"
import torch

from transformers import Qwen2_5OmniProcessor
from vllm import LLM, SamplingParams
from qwen_omni_utils import process_mm_info

# vLLM engine v1 not supported yet
os.environ['VLLM_USE_V1'] = '0'

MODEL_PATH = "/data/pretrained_models/Qwen2.5-Omni-7B"

llm = LLM(
    model=MODEL_PATH, trust_remote_code=True, gpu_memory_utilization=0.9,
    tensor_parallel_size=torch.cuda.device_count(),
    limit_mm_per_prompt={'image': 0, 'video': 0, 'audio': 0},
    seed=1234,
    max_model_len=1024,
)

sampling_params = SamplingParams(
    temperature=0.7,
    max_tokens=1024,
)

processor = Qwen2_5OmniProcessor.from_pretrained(MODEL_PATH)

messages = [
    {
        "role": "system",
        "content": "You are Qwen, a virtual human developed by the Qwen Team, Alibaba Group, capable of perceiving auditory and visual inputs, as well as generating text and speech.",
    },
    {
        "role": "user",
        "content": [
           {"type": "text", "text": "你是谁？请介绍一下你自己"},
        ],
    },
]

text = processor.apply_chat_template(
    messages,
    tokenize=False,
    add_generation_prompt=True,
)

audios, images, videos = process_mm_info(messages, use_audio_in_video=True)

inputs = {
    'prompt': text[0],
    # 'multi_modal_data': {},
    "mm_processor_kwargs": {
        "use_audio_in_video": True,
    },

}

if images is not None:
    inputs['multi_modal_data']['image'] = images
if videos is not None:
    inputs['multi_modal_data']['video'] = videos
if audios is not None:
    inputs['multi_modal_data']['audio'] = audios

print('-'*100)
print(inputs)
print('-'*100)

outputs = llm.generate(inputs, sampling_params=sampling_params)
print('-'*100)
print(outputs)
print('-'*100)
print(outputs[0].outputs[0].text)
print('*'*100)

得到下面输出：
[RequestOutput(request_id=0, prompt='<|im_start|>system\nYou are Qwen, a virtual human developed by the Qwen Team, Alibaba Group, capable of perceiving auditory and visual inputs, as well as generating text and speech.<|im_end|>\n<|im_start|>user\n你是谁？请介绍一下你自己<|im_end|>\n<|im_start|>assistant\n', prompt_token_ids=[151644, 8948, 198, 2610, 525, 1207, 16948, 11, 264, 4108, 3738, 7881, 553, 279, 1207, 16948, 7909, 11, 54364, 5737, 11, 12875, 315, 817, 46344, 82529, 323, 9124, 11127, 11, 438, 1632, 438, 23163, 1467, 323, 8806, 13, 151645, 198, 151644, 872, 198, 105043, 100165, 11319, 14880, 109432, 107828, 151645, 198, 151644, 77091, 198], encoder_prompt=None, encoder_prompt_token_ids=None, prompt_logprobs=None, outputs=[CompletionOutput(index=0, text='', token_ids=(151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872, 151872), cumulative_logprob=None, logprobs=None, finish_reason=length, stop_reason=None)], finished=True, metrics=RequestMetrics(arrival_time=1743668816.4981759, last_token_time=1743668825.3031282, first_scheduled_time=1743668816.4996989, first_token_time=1743668816.5261886, time_in_queue=0.0015230178833007812, finished_time=1743668825.3033078, scheduler_time=0.029592805542051792, model_forward_time=None, model_execute_time=None, spec_token_acceptance_counts=[0]), lora_request=None, num_cached_tokens=0, multi_modal_placeholders={})]

**Comment by wangxiongts** [comment]
我们这两天会在transformers的代码merge好之后，重新发布一次镜像，目前vllm代码还在持续迭代，会在发布的新镜像里修复

**Comment by wangxiongts** [comment]
目前新的镜像和vllm已经发布，可以在readme里查看
